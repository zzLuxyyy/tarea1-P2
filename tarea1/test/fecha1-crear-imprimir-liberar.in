crearFecha 25/8/2025
imprimirFecha
liberarFecha
Fin