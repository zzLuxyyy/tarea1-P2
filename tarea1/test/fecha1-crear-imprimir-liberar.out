1>2>25/8/2025
3>4>Fin.
